examples/js/exporters/GLTFExporter.js

```diff
@@ -964,12 +985,30 @@ THREE.GLTFExporter.prototype = {
 
 			}
 
-			// pbrMetallicRoughness.baseColorTexture
+			// pbrMetallicRoughness.baseColorTexture or pbrSpecularGlossiness diffuseTexture
 			if ( material.map ) {
 
 				var baseColorMapDef = { index: processTexture( material.map ) };
 				applyTextureTransform( baseColorMapDef, material.map );
-				gltfMaterial.pbrMetallicRoughness.baseColorTexture = baseColorMapDef;
+
+				if ( material.isGLTFSpecularGlossinessMaterial ) {
+
+					gltfMaterial.extensions.KHR_materials_pbrSpecularGlossiness.diffuseTexture = baseColorMapDef;
```

Comment:
I think we should set `KHR_materials_pbrSpecularGlossiness.diffuseTexture` *AND* `pbrMetallicRoughness.baseColorTexture` for a spec/gloss material. That way engines that don't support the spec/gloss extension will still get base color.